splunklib/client.py

```diff
@@ -3740,18 +3747,20 @@ def data(self):
         """
         return KVStoreCollectionData(self)
 
-    def update_index(self, name, value):
```

Comment:
I think we want to add support for accelerated field indexes without removing current index support

Reply:
In Splunk Enterprise v6.4 archive docs we found that for kvstore `index.<indexName>` property exist but in later version it was updated to `accelerated_field.<field_name>`, so due to this change we updated the same method. PFA docs screenshots for reference, let us know if we still want existing index support.

Splunk v6.4 docs:
![splunkv6 4_doc](https://user-images.githubusercontent.com/84171610/211742036-b8ac4653-c609-4bcb-82cd-60b1b38f6bdf.png)

Splunk v8.2 docs:
![splunkv8 2_doc](https://user-images.githubusercontent.com/84171610/211742186-aa812af9-8252-4ac4-b9bb-3164238d14a9.png)

Reply:
OK great thanks for clarifying, in that case the changes look good to me 👍